hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/handlers/EthereumTransactionHandler.java

```diff
@@ -161,6 +161,19 @@ public void handle(@NonNull final HandleContext context) throws HandleException
         throwIfUnsuccessful(outcome.status());
     }
 
+    /**
+     * Does work needed to externalize details after an Ethereum transaction is throttled.
+     * @param context the handle context
+     */
+    public void handleThrottled(@NonNull final HandleContext context) {
+        final var component = provider.get().create(context, ETHEREUM_TRANSACTION);
+        final var ethTxData =
+                requireNonNull(requireNonNull(component.hydratedEthTxData()).ethTxData());
```

Comment:
(first time I've seen `requireNonNull` nested like this ... it's allowed (obviously) ... but ugly.  but the need to pass nulls along in a chain of method calls is definitely needed - happens all the time.  Java needs the C# `??` "null-coalescing operator".)

Reply:
Yes that would be nice...